app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.help.faq.FAQListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/FAQs" />
```

Comment:
We should have a string that contextualizes the screen like we do elsewhere, e.g. "Frequently asked questions screen".

Reply:
@Sparsh1212 As per #2743 replace the label as `FAQs (Frequently Asked Questions)`.
You can create a string something like `faq_activity_label`

Reply:
Done